includes/class-sensei-settings.php

```diff
@@ -219,6 +219,14 @@ public function init_fields () {
 								'section' => 'default-settings'
 								);
 
+		$fields['html_security_disable'] = array(
```

Comment:
rename this to something intention-revealing, maybe `sensei_video_embed_html_sanitization_disable`